examples/visualization/plot_study.ipynb

```diff
@@ -4,6 +4,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](http://colab.research.google.com/github/optuna/optuna/blob/master/examples/visualization/plot_study.ipynb)\n",
+    "It's recommended to right-click \"Open In Colab\" to open this notebook if you are on GitHub.\n",
+    "\n",
```

Comment:
Thanks for the additional comment. This is maybe a nitpick but I'm wondering if we shouldn't mention that it's necessary to open it in a new tab/window (you don't necessarily have to right-click to do this depending on the browser). Something along "Colab must be opened in a new tab or window if you are on GitHub."?